interfaces/connection_test.go

```diff
@@ -329,3 +329,29 @@ func (s *connSuite) TestCopyAttributes(c *C) {
 	cpy["e"].(map[string]interface{})["e1"] = "x"
 	c.Check(orig["e"].(map[string]interface{})["e1"], Equals, "E1")
 }
+
+func (s *connSuite) TestNewConnectedPlugExplicitStaticAttrs(c *C) {
+	staticAttrs := map[string]interface{}{
+		"baz": "boom",
+	}
+	dynAttrs := map[string]interface{}{
+		"foo": "bar",
+	}
+	plug := NewConnectedPlug(s.plug, staticAttrs, dynAttrs)
+	c.Assert(plug, NotNil)
+	c.Assert(plug.StaticAttrs(), DeepEquals, staticAttrs)
+	c.Assert(plug.DynamicAttrs(), DeepEquals, dynAttrs)
```

Comment:
Is it supposed to be the exact same value, or a copy?  In either case, I suggest testing the behavior here.